Send Birthday data to Salesforce

funmark opened the pull request:
The donor birthday is not being sent to salesforce, because it was not
available.  Now that it is available, this story sends the field to
salesforce.

Resolves:  FUN-4554

funmark committed:
Send Birthday data to Salesforce

The donor birthday is not being sent to salesforce, because it was not
available.  Now that it is available, this story sends the field to
salesforce.

Resolves:  FUN-4554

funmark committed:
Birthdate Field To Filename Mismatch

Metadata filenames must match json field names. This commit names
the birthdate filename accordingly.

funmark committed:
Cleanup

Alexalex1432 commented on src/package.xml:
```diff
@@ -29,7 +29,75 @@
+        <members>frField.amount</members>
+        <members>frField.annualDonations</members>
+        <members>frField.anonymous</members>
+        <members>frField.campaignGoalId</members>
```
Are we missing birth date in this list? assuming alphabetical it should be above this line

Alexalex1432 replied:
I'm also fine if we change this back to a wildcard, because we know that we manage all of these records and we don't have to worry about adding new ones (like now where birth date is missing)

Alexalex1432 commented on salesforce.schema:
```diff
@@ -0,0 +1 @@
+place holder
```
did you add this or was this from the IDE?

funmark replied:
IDE

funmark commented:
IDE

*Mark Dufresne*

*Staff Software Engineer*

<https://www.funraise.io/>

<https://www.facebook.com/funraiseplatform/>
<https://www.instagram.com/funraise/>
<https://www.linkedin.com/company/funraise/>   <https://blog.funraise.io/>
<https://twitter.com/funraise>

On Mon, Aug 13, 2018 at 11:39 AM, Alex Molina <notifications@github.com>
wrote:

> *@Alexalex1432* commented on this pull request.
> ------------------------------
>
> In salesforce.schema
> <https://github.com/funraise/Funraise-Salesforce/pull/8#discussion_r209715313>
> :
>
> > @@ -0,0 +1 @@
> +place holder
>
> did you add this or was this from the IDE?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/funraise/Funraise-Salesforce/pull/8#pullrequestreview-145783259>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AnN8kwvBMljRQMNYs_b6etsoln9KYnVXks5uQcfhgaJpZM4VyyN5>
> .
>

Alexalex1432 reviewed (changes requested):
add birthdate to package.xml

funmark committed:
Undo IDE auto formatting

IDE expanded wildcards, so need to undo wildcard expansion

funmark committed:
Ignore IDE generated project files

Eclipse or Salesforce plugin generates a schema file that we do not
want version controlled.

funmark committed:
Ignore IDE generated files

Eclipse IDE generates salesforce.schema, which we do not want to
version control.
